Added an argument for the use of cloud providers (#151)
SW709E19/Report: report/PartDesign/ChapterPrototype/Prototype.tex

The communication between sensor devices, gateway, and cloud was managed by using Mosquitto \cite{Mosquitto}, a message broker for the MQTT protocol which can be configured to bridge messages to a remote broker.  

\subsection{Cloud architecture}

> For the cloud architecture a shared cloud provider was chosen for multiple reasons.

Author: Det hedder en shared når vi er på Google, Microsoft og lignende. 
Det er en private cloud provider når vi hoster den selv og derfor ikke skal dele den med andre.

The communication between sensor devices, gateway, and cloud was managed by using Mosquitto \cite{Mosquitto}, a message broker for the MQTT protocol which can be configured to bridge messages to a remote broker.  

\subsection{Cloud architecture}
For the cloud architecture a cloud provider was chosen for multiple reasons.

> Firstly the group had previously used the internet services at the AAU (ITS) for hosting student projects, but not always with the greatest success, mostly due to the extra work of maintaining those services.

Reviewer: ```suggestion
While it was possible to use the services offered by AAU IT Services to host the project, some group members had previously used these services and not with great success, mostly due to the extra work of maintaining those services.
```

> Lastly the added complexity of developing the backend for handling IoT devices would take much time from other aspects for the project.

Reviewer: ```suggestion
Lastly the added complexity of developing the backend for handling IoT devices would take much time from other aspects of the project.
```

> If an existing cloud provider also could provide a backend for handling the different devices it would be beneficial for the project as a whole.

Reviewer: Jeg er ikke helt sikker på at jeg forstår det her 100%, men jeg har også en meget begrænset forståelse af GCP. Kan det tænkes at læseren kan opleve samme problem? Kan vi uddybe eller forklare det lidt bedre?

> Since an existing shared cloud provider also provides a backend for handling the different devices which is beneficial for the project as a whole how ever it might introduce some vendor lock in that should be avoided in the final system.

Reviewer: ```suggestion
A cloud provider also provides a backend for handling the different devices which is beneficial for the project as a whole, however it might introduce vendor lock-in which should be avoided in the final system.
```
